code/platformio.ini

```
@@ -1006,3 +1006,7 @@ src_build_flags = -DKINGART_CURTAIN_SWITCH
 [env:aoycocr-x5p]
 extends = env:esp8266-1m-base
 src_build_flags = -DAOYCOCR_X5P
+
+[env:hugoai-awp02l-n]
+extends = env:esp8266-1m-base
+src_build_flags = -HUGOAI_AWP02L_N
```

shouldn't this be prefixed with -D ?
```suggestion
src_build_flags = -DHUGOAI_AWP02L_N
```